app/models/audit/event/password.rb

```diff
@@ -0,0 +1,61 @@
+module Audit
+  module Event
+    class Password
+
+      def initialize(
+        operation: 'change',
+        user: ,
+        success:,
+        error_message: nil
+      )
```

Comment:
I know it doesn't matter from the caller's perspective since we're using keyword args, but I'd put all the optional params at the end just for consistency / readability.

Also nit: I think lines 5-10 can fit on a single line under 80 chars.

Reply:
Oh also: If there are other operations here, I think I might prefer making the callers _always_ be explicit about, rather than having a default value for the most common case.  That way, readers can glean everything they know from the calling site, rather than having to jump to definition to see the operation.

Wdyt?

Reply:
So I put this in the draft to let it simmer for a minute in my mind. I think I'm actually inclined to remove `operation` from the inputs, because the only operation we know of right now for passwords is `change`.

The mental exercise here was understanding how it could be parameterized, since that was one of the immediate hurdles when trying to split the single `authn` event into `authn`, `login`, and `inject_client_cert` in the earlier PR.

I think it would be pretty obvious in the current structure how to make this an option of the caller, if that became a requirement in the future. Although, I think the current pattern with `authn` would lead to new operations of `password` just becoming classes themselves.

Reply:
I agree.  YAGNI wins especially when the future change, should we need it, will be trivial to make.